src/modal-comment.js

```diff
@@ -0,0 +1,206 @@
+import {
+  DataAPI,
+} from './Involvement_API';
+
+const modalContainer = document.getElementById('modal-popup-container');
+const headerContainer = document.getElementById('navContainer');
+const mainContainer = document.getElementById('mainContainer');
+const footeContainer = document.getElementById('footerContainer');
+const countComments = (pokeId) => DataAPI.microverseInvolvement.getComments(pokeId).then(
+  (response) => response.length,
+);
```

Comment:
- [ ] The method that you have used for the counting comments and method that you have tested in the `CounterComments.test.js` are not the same methods. You should test the method that you have used for counting the comments.

Reply:
Thank you for your feedback @sinansevgi. The amends are made it in commit c1bc562.